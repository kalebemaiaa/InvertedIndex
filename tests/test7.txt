QueryOr(QueryWord("embaixador"), QueryAnd(QueryWord("consulado"), QueryWord("secretário")))
1208.raw
1256.raw
1441.raw
532.raw
3426.raw
189.raw
5215.raw
4212.raw
5483.raw
3727.raw
4266.raw
1938.raw
11416.raw
6022.raw
1275.raw
2035.raw
1523.raw
2513.raw
12010.raw
2080.raw
10772.raw
5093.raw
38.raw
11608.raw
1873.raw
11005.raw
11072.raw
3942.raw
2005.raw
5014.raw
894.raw
11606.raw
4573.raw
6289.raw
5426.raw
3476.raw
11431.raw
632.raw
10759.raw
4096.raw
1949.raw
3317.raw
3386.raw
3102.raw
2365.raw
12049.raw
11049.raw
171.raw
8883.raw
6239.raw
3379.raw
11516.raw
5946.raw
11152.raw
2809.raw
3482.raw
11216.raw
4118.raw
4510.raw
4934.raw
11066.raw
11801.raw
11784.raw
5865.raw
4010.raw
92.raw
2092.raw
1130.raw
1679.raw
11549.raw
3977.raw
5514.raw
2923.raw
4338.raw
4261.raw
5714.raw
443.raw
151.raw
5599.raw
3711.raw
266.raw
3177.raw
79.raw
3119.raw
2043.raw
6227.raw
4205.raw
3015.raw
2102.raw
4054.raw
4449.raw
11114.raw
4363.raw
5330.raw
11406.raw
908.raw
5501.raw
1378.raw
4482.raw
1400.raw
180.raw
4741.raw
1127.raw
11841.raw
1615.raw
11969.raw
3931.raw
11223.raw
11838.raw
1017.raw
2867.raw
4956.raw
4244.raw
3320.raw
2.raw
10911.raw
1553.raw
1977.raw
5085.raw
1936.raw
5055.raw
6222.raw
3202.raw
11196.raw
4941.raw
1568.raw
8751.raw
569.raw
5718.raw
4479.raw
4214.raw
4747.raw
5811.raw
8827.raw
1301.raw
299.raw
11521.raw
3877.raw
1503.raw
4647.raw
11782.raw
789.raw
1808.raw
598.raw
11730.raw
11197.raw
1111.raw
11909.raw
5729.raw
2119.raw
4253.raw
5398.raw
5962.raw
4058.raw
8730.raw
1980.raw
3093.raw
11592.raw
3311.raw
4950.raw
2094.raw
5031.raw
2822.raw
8895.raw
1857.raw
3547.raw
3842.raw
3499.raw
3083.raw
11263.raw
4276.raw
10965.raw
656.raw
3040.raw
529.raw
2228.raw
4319.raw
1695.raw
10860.raw
11346.raw
1534.raw
4844.raw
6048.raw
623.raw
4104.raw
4195.raw
931.raw
10820.raw
5516.raw
483.raw
5941.raw
1836.raw
4111.raw
10968.raw
5504.raw
5454.raw
1198.raw
3325.raw
3686.raw
4676.raw
