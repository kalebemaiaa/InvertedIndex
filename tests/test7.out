672
1090
1546
1646
1739
1828
2151
2332
2412
3051
3588
4231
4410
5012
5121
